cccatalog-api/cccatalog/urls.py

```diff
@@ -41,7 +45,7 @@
 an API key as soon as possible. Authorized clients have a higher rate limit
 of 10000 requests per day and 100 requests per minute. Additionally, Creative
 Commons can give your key an even higher limit that fits your application's
-needs. See the `/oauth2/register` endpoint for instructions on obtaining
+needs. See the `/v1/oauth2/register` endpoint for instructions on obtaining
```

Comment:
The API audit task mentioned replacing this endpoint with `/v1/auth_tokens/` for RESTfulness – why didn't you end up doing that?

Reply:
It didn't end up in the spec somehow. I changed the URLs to `/v1/auth_tokens/`.